recipes-bsp/grub/grub-nilrt.inc

```diff
@@ -5,6 +5,7 @@ SRCREV = "832e7862089a7cbc985ae09a17f6827de6c85bf8"
 
 SRC_URI = "\
 	${NILRT_GIT}/grub.git;branch=${GRUB_BRANCH} \
+	file://x86-64-Treat-R_X86_64_PLT32-as-R_X86_64_PC32.patch \
```

Comment:
This is fine for now. Can you file a tech debt item to rebase our grub branch upon something more recent.

Reply:
Work [item](https://ni.visualstudio.com/DevCentral/_workitems/edit/1119988) created.